adafruit_bitmap_font/bitmap_font.py

```diff
@@ -56,4 +58,11 @@ def load_font(filename: str, bitmap: Optional[Bitmap] = None) -> Union[bdf.BDF,
 
         return ttf.TTF(font_file, bitmap)
 
+    if (
+        filename.endswith("bin") or filename.endswith("lvfontbin")
+    ) and first_four == LVGL_HEADER_SIZE:
```

Comment:
`LVGL_HEADER_SIZE` needs to be imported in this file. I added `from .lvfontbin import LVGL_HEADER_SIZE` on my device and it allows the new font loader to work

Reply:
I moved it instead. The lvfontbin file shouldn't need it.